### ISSUE TITLE
Video Processing is very slow, frames are missing while counting persons

### ISSUE THREAD

**Issue by Jitesh-Mohite** [ahmetozlu/tensorflow_object_counting_api#34]
How we can increase FPS that frames should not get missed while live feeding from the webcam. 
Is there a thread mechanism we should use for it. 

file: real_time_counting

**Comment by Jitesh-Mohite** [comment]
@ahmetozlu: Can you please help us with this?

**Comment by Jitesh-Mohite** [comment]
@ahmetozlu : do you get the chance to look into it? it will be good for us if you guide.

**Comment by ahmetozlu** [comment]
[**_ssd_mobilenet_coco_v1_** ](https://gtgit.fw.garanti.com.tr/projects/JAVA/repos/gar_svc_sms_keyword/browse/gar-svc-sms-keyword-service-layer?at=refs%2Fheads%2Ffeature%2Fsms-keyword-dev-62040252)is the fastest option you can run among **_[TensorFlow models zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md#coco-trained-models)_**. You should use **_[TensorFlow GPU](https://www.tensorflow.org/install/gpu)_** to run the detection model faster.

You should perform "transfer learning" or "fine-tuning" to increase the accuracy for your case!
